packages/compile-solidity/test/test_supplier.js

```diff
@@ -170,10 +168,24 @@ describe("CompilerSupplier", function () {
         options: cachedOptions
       });
 
-      assert(await fse.exists(expectedCache), "Should have cached compiler");
+      const cachedCompilerFilenames = fse.readdirSync(compilerCacheDirectory);
+      const compilerFilename = cachedCompilerFilenames.find(filename => {
+        return filename.includes("v0.4.21+commit.dfe3193c");
+      });
+
+      assert.equal(
```

Comment:
(Hey @eggplantzzz looks like you inverted this one)

Reply:
oh I did this algorithmically and didn't think about the value - `isUndefined` should be used

Reply:
oh no that's also wrong, it should be defined :P

Reply:
Well, you're just using `assert` and not `chai` here, right?  Node's builtin `assert` doesn't have `isDefined`, that's a `chai` thing...

Reply:
oh I thought it was using chai...easily remedied